src/devices/machine/sis7001_usb.cpp

```diff
@@ -0,0 +1,93 @@
+// license: BSD-3-Clause
+// copyright-holders: Angelo Salese
+/**************************************************************************************************
+
+	SiS 7001 USB Host controller
+
+    TODO:
+	- Stub interface, to be improved;
+	- PCI values omitted from docs, assumes same as OpenHCI;
+
+**************************************************************************************************/
+
+#include "emu.h"
+#include "sis7001_usb.h"
+
+#define LOG_IO     (1U << 1) // log PCI register accesses
+#define LOG_MAP    (1U << 2) // log full remaps
+
+#define VERBOSE (LOG_GENERAL | LOG_IO | LOG_MAP)
+//#define LOG_OUTPUT_FUNC osd_printf_warning
+
+#include "logmacro.h"
+
+#define LOGIO(...)     LOGMASKED(LOG_IO,   __VA_ARGS__)
+#define LOGMAP(...)    LOGMASKED(LOG_MAP,  __VA_ARGS__)
+
+DEFINE_DEVICE_TYPE(SIS7001_USB, sis7001_usb_device, "sis7001_usb", "SiS 7001 USB Host Controller")
+
+sis7001_usb_device::sis7001_usb_device(const machine_config &mconfig, const char *tag, device_t *owner, uint32_t clock)
+	: pci_device(mconfig, SIS7001_USB, tag, owner, clock)
+
+{
+
+}
+
+void sis7001_usb_device::device_add_mconfig(machine_config &config)
+{
+
+}
+
+void sis7001_usb_device::config_map(address_map &map)
+{
+	pci_device::config_map(map);
+}
+
+void sis7001_usb_device::io_map(address_map &map)
+{
+	// operational mode
+	map(0x000, 0x000).lr8(NAME([]() { return 0x00000110; }));
```

Comment:
This isn't returning an 8-bit value like it says.

Reply:
Good catch.